### PR TITLE
Prow: Cleanup old config

### PR DESCRIPTION
**Pull request by lentzi90**
Removed old config files and synced them to the new location.

### CODE REVIEW

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -1100,6 +1100,22 @@ presubmits:
...
+          - args:
+              - ./hack/markdownlint.sh
+            command:
+              - sh
+            env:
+              - name: IS_CONTAINER
+                value: "TRUE"
+            image: docker.io/pipelinecomponents/markdownlint:0.12.0@sha256:0b8f9fcf0410257b2f3548f67ffe25934cfc9877a618b9f85afcf345a25804a2
+            imagePullPolicy: Always
+
```
Comment (lentzi90) [inline prow/manifests/overlays/metal3/config.yaml:1118]: This was added to the old config after I copied them to the new location. Just synced them before deleting the old config.

**Review by tuminoid — APPROVED** [review]
/lgtm

**Review by kashifest — APPROVED** [review]

**Pull request merged**